Signal/src/ViewControllers/ShowGroupMembersViewController.m

```diff
@@ -280,37 +296,43 @@ - (void)showContactInfoViewForRecipientId:(NSString *)recipientId
         [self presentViewController:alertController animated:YES completion:nil];
         return;
     }
-
+    
+    CNContactViewController *_Nullable contactViewController;
     if (signalAccount) {
-        // FIXME This is broken until converted to Contacts framework.
-        ABPersonViewController *view = [[ABPersonViewController alloc] init];
-
-        ABAddressBookRef addressBookRef = ABAddressBookCreateWithOptions(NULL, nil);
-        // Assume person is already defined.
-        view.displayedPerson = ABAddressBookGetPersonWithRecordID(addressBookRef, signalAccount.contact.recordID);
-        view.allowsActions = NO;
-        view.allowsEditing = YES;
+        CNContact *_Nullable cnContact = signalAccount.contact.cnContact;
+        if (cnContact) {
+            contactViewController = [CNContactViewController viewControllerForContact:cnContact];
+        }
+    }
 
-        [self.navigationController pushViewController:view animated:YES];
-    } else {
-        // FIXME This is broken until converted to Contacts framework.
-        ABUnknownPersonViewController *view = [[ABUnknownPersonViewController alloc] init];
+    if (!contactViewController) {
+        CNMutableContact *newContact = [CNMutableContact new];
+        CNPhoneNumber *phoneNumber = [CNPhoneNumber phoneNumberWithStringValue:recipientId];
+        CNLabeledValue<CNPhoneNumber *> *labeledPhoneNumber = [CNLabeledValue labeledValueWithLabel:CNLabelPhoneNumberMain
+                                                                             value:phoneNumber];
+        newContact.phoneNumbers = @[labeledPhoneNumber];
+        
+        contactViewController = [CNContactViewController viewControllerForNewContact:newContact];
+    }
 
-        ABRecordRef aContact = ABPersonCreate();
-        CFErrorRef anError   = NULL;
+    contactViewController.delegate = self;
+    contactViewController.allowsActions = NO;
+    contactViewController.allowsEditing = YES;
+    contactViewController.navigationItem.leftBarButtonItem =
```

Comment:
We're going to let users edit contacts from other places (like 1:1 conversation settings) too, right? I wonder if we should move this to a helper class (e.g. `ViewControllerUtils`)?